requirements.txt

```diff
@@ -10,3 +10,4 @@ Yapsy==1.11.223
 configobj==4.7.0
 redis>=2.10.5
 morph==0.1.2
+ibm_db>=2.0.7
```

Comment:
Thank you for advice. I will modify the code according your suggestion. If I finish it, I create new PR.